src/test/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseGroupEndpointUnitTest.java

```diff
@@ -7,9 +7,7 @@
 import static org.junit.Assert.assertThat;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.handler;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
```

Comment:
The checkstyle might complain about the wildcard import. For some reason the intellij plugin doesn't fix the import style. You have to follow the bottom part of the section in https://github.com/google/google-java-format to do that

Reply:
Fixed